packages/ripple-nuxt-ui/lib/templates/plugin.js

```diff
@@ -14,55 +14,55 @@ let rplOptions = {
 }
 
 <% if (options.hostname) { %>
-  rplOptions.hostname = options.hostname
+rplOptions.hostname = options.hostname
 <% } %>
 
 <% if (options.quickexit) { %>
   rplOptions.quickexit = options.quickexit
-<% } %>
+    <% } %>
 
 <% if (options.quickexiturl) { %>
   rplOptions.quickexiturl = options.quickexiturl
-<% } %>
+    <% } %>
 
 <% if (typeof options.viclogo !== 'undefined') { %>
   rplOptions.viclogo = options.viclogo
-<% } %>
+    <% } %>
 
 <% if (typeof options.imgQueryString !== 'undefined') { %>
-    rplOptions.imgQueryString = options.imgQueryString
-<%   } else { %>
-    rplOptions.imgQueryString = (bp) => {
-      // Provides query string for images using section kraken.io - https://www.section.io/docs/modules/kraken/reference/kraken-advanced-config/
-      const params = {
-        strategy: 'auto'
-      }
-      for (const key in bp) {
-        const whitelistParams = ['height', 'width', 'strategy']
-        if (bp.hasOwnProperty(key) && whitelistParams.includes(key)) {
-          params[key] = bp[key]
-        }
+  rplOptions.imgQueryString = options.imgQueryString
+    <%   } else { %>
+  rplOptions.imgQueryString = (bp) => {
+    // Provides query string for images using section kraken.io - https://www.section.io/docs/modules/kraken/reference/kraken-advanced-config/
+    const params = {
+      strategy: 'auto'
+    }
+    for (const key in bp) {
+      const whitelistParams = ['height', 'width', 'strategy']
+      if (bp.hasOwnProperty(key) && whitelistParams.includes(key)) {
+        params[key] = bp[key]
       }
-      return '?' + queryString(params)
     }
-<% } %>
+    return '?' + queryString(params)
+  }
+  <% } %>
 
 <% if (typeof options.viclogoFooter !== 'undefined') { %>
-  rplOptions.viclogoFooter = options.viclogoFooter
-<% } %>
+    rplOptions.viclogoFooter = options.viclogoFooter
+      <% } %>
 
 <% if (options.card && options.card.trimFieldfonts) { %>
-  rplOptions.card = rplOptions.card || {}
-  rplOptions.card.trimFieldfonts = options.card.trimFieldfonts
-<% } %>
+    rplOptions.card = rplOptions.card || {}
+    rplOptions.card.trimFieldfonts = options.card.trimFieldfonts
+      <% } %>
 
 <% if (options.externalLinksInNewWindow) { %>
-  rplOptions.externalLinksInNewWindow = options.externalLinksInNewWindow
-<% } %>
+    rplOptions.externalLinksInNewWindow = options.externalLinksInNewWindow
+      <% } %>
 
 <% if (options.customIcon) { %>
-  // Add custom icons to library.
-  addCustomIcons(require.context('../assets/ripple-icon/', true, /\.svg$/))
-<% } %>
+    // Add custom icons to library.
+    addCustomIcons(require.context('../assets/ripple-icon/', true, /\.svg$/))
```

Comment:
Should this be changed to load the js?

Reply:
Yes, JS will be loaded automatically as they are imported in `Atoms/Icon/index.js`
```
import './icons/index.js'
```

Reply:
@tim-yao, I added below require.context which seems to work fine in my local.